wpiunits/src/main/java/edu/wpi/first/units/Units.java

```diff
@@ -250,6 +250,23 @@ private Units() {
   public static final Mult<Mult<Mass, Distance>, Distance> KilogramSquareMeters =
       Kilograms.mult(Meters).mult(Meters);
 
+  // Force
+  /** The base SI unit for force. */
+  public static final Mult<Mass, Velocity<Velocity<Distance>>> Newtons =
+      Kilograms.mult(Meters.per(Second).per(Second));
```

Comment:
Giving this a name other than the default (which would be "Kilogram-Meter per Second per Second") would be good. Might want to add onto `UnitBuilder` to make aliasing easier.

Reply:
I wanted to check first before adding more units.

I’m not sure what you are saying to modify?

Reply:
Never mind I think I take your meaning.